src/main/java/frc/robot/commands/ExtendCoralArm.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.CoralArm;
+
+public class ExtendCoralArm extends Command {
+    private CoralArm coralArm;
```

Comment:
```suggestion
    private final CoralArm coralArm;
```
finalize where possible